transports/janus_websockets.c

```diff
@@ -618,6 +621,12 @@ int janus_websockets_init(janus_transport_callbacks *callback, const char *confi
 			item = janus_config_get(config, config_general, janus_config_type_item, "ws_ip");
 			if(item && item->value) {
 				ip = (char *)item->value;
+#ifdef __FreeBSD__
+				struct in_addr addr;
+				if(inet_net_pton(AF_INET, ip, &addr, sizeof(addr))>0) {
```

Comment:
Code style: there should be spaces before and after the `>` comparator.
Besides, since the action is simple, I don't think it's needed to wrap it in curly brackets.

Reply:
done